### PR TITLE
Temperature unit conversion

### PR DESCRIPTION
**Pull request by anant00729**
To test:

1. Open http://localhost:3000/sensor/<location_id>/readings page.
2. check the temperature unit to be C unit and the values should be between 10 and 30 degree C.
3. Go to http://localhost:3000/farm. and change the units from "Metric" to "Imperial".
4. Go back to http://localhost:3000/sensor/<location_id>/readings page.
5. check the temperature unit to be F unit and the values should be between 60 and 85 degree F.

Before: 

Screen Shot 2022-08-16 at 2 26 42 PM

After: 

Screen Shot 2022-08-16 at 2 27 22 PM

### CODE REVIEW

File: packages/webapp/src/containers/SensorReadings/saga.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ import {
 import { sensorUrl } from '../../apiConfig';
 import { getHeader } from '../../containers/saga';
 import { findCenter } from './utils';
-import { AMBIENT_TEMPERATURE, CURRENT_DATE_TIME } from './constants';
+import { CURRENT_DATE_TIME } from './constants';
+import { getTemperatureValue } from '../../components/Map/PreviewPopup/utils.js';
+import { TEMPERATURE } from './constants';
```
Comment (seinnlee) [inline packages/webapp/src/containers/SensorReadings/saga.js:32]: Can we put these constants on line 21? Both `CURRENT_DATE_TIME` and `TEMPERATURE` are being imported from other constants on line 21

**Review by seinnlee — APPROVED** [review]
It looks good to me. Just minor comments about some import statements. Other than that, it looks functionally good.

File: packages/webapp/src/containers/SensorReadings/saga.js (hunk 1 above)
Comment (anant00729) [reply packages/webapp/src/containers/SensorReadings/saga.js]: Done

**Review by Cali0707 — APPROVED** [review]
LGTM

**Comment by fsayyid** [comment]
@anant00729 please deal with merge conflicts

**Comment by anant00729** [comment]
Synced integration with LF-2732/temperature-unit-conversion. Please check now.

**Pull request merged**